lighthouse-core/report/html/report-styles.css

```diff
@@ -186,8 +190,11 @@
     --report-border-color-secondary: var(--color-gray-200);
     --report-text-color-secondary: var(--color-gray-400);
     --report-text-color: var(--color-gray-100);
-    --topbar-background-color: var(--color-gray);
     --snippet-color: var(--color-cyan-500);
+    --topbar-background-color: var(--color-gray);
+  	--toplevel-warning-background-color: #544B40;
```

Comment:
not sure if github is playing tricks on me but i think we have some tabs here, can you replace with spaces?

Reply:
My bad, fixed now